tests/test_main_run_main.py

```
@@ -0,0 +1,460 @@
+# =============================================================================
+
+
+@patch("asyncio.run")
+@patch("mmrelay.config.load_config")
+@patch("mmrelay.config.set_config")
+@patch("mmrelay.log_utils.configure_component_debug_logging")
+@patch("mmrelay.main.print_banner")
+def test_run_main(
+    mock_print_banner,
+    mock_configure_debug,
+    mock_set_config,
```

Ruff (0.15.12)

[warning] Unused function argument: `mock_set_config`

(ARG001)